app/code/Magento/Sales/Test/Unit/ViewModel/CreditMemo/Create/ItemsToRenderTest.php

```diff
@@ -126,18 +127,39 @@ public function testGetItems(): void
             ->willReturn($this->creditmemo);
         $this->creditmemo->method('getStoreId')
             ->willReturn(1);
+
+        $this->orderItem->method('getQtyInvoiced')
+            ->willReturn($qtyInvoiced);
+        $this->orderItem->method('getQtyRefunded')
+            ->willReturn($qtyRefunded);
+
         $this->creditmemoItem->method('getOrderItem')
             ->willReturn($this->orderItem);
-        $this->orderItem->method('getParentItem')
-            ->willReturn($this->orderItemParent);
-        $this->orderItemParent->method('getItemId')
-            ->willReturn(1);
-        $this->converter->method('itemToCreditmemoItem')
-            ->willReturn($this->creditmemoItemParent);
 
-        $this->assertEquals(
-            [$this->creditmemoItemParent, $this->creditmemoItem],
-            $this->itemsToRender->getItems()
-        );
+        $items = $this->itemsToRender->getItems();
+
+        $this->assertCount($expectedCount, $items);
+
+        if ($shouldContainItem) {
+            $this->assertContains($this->creditmemoItem, $items);
+        } else {
+            $this->assertNotContains($this->creditmemoItem, $items);
+        }
+    }
+
+    /**
+     * Data provider for testing the invoicing status.
+     *
+     * @return array
+     */
+    public static function itemInvoicingDataProvider(): array
+    {
+        return [
+            // Test case: Item is invoiced, should be included
+            [1.0, 0.0, 1, true],
+
+            // Test case: Item is not invoiced, should not be included
+            [0.0, 0.0, 0, false],
```

Comment:
can we have names for these cases instead of writing it comments?

I guess something like this should work:
```suggestion
            'Item is invoiced, should be included' => [1.0, 0.0, 1, true],
            'Item is not invoiced, should not be included' => [0.0, 0.0, 0, false],
```